recipe/meta.yaml

```
@@ -27,7 +27,7 @@ source:
     - patches/0018-Use-sed-from-system.patch
 
 build:
-  skip: true  # [win]
+  skip: true  # [win or not (linux and x86_64)]
```

`win` is included in `not (linux and x86_64)` isn't it?
Unless you want to specifically mention `win` for $REASONS.

```suggestion
  skip: true  # [not (linux and x86_64)]
```